staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -102,7 +102,7 @@ type KubeSchedulerConfiguration struct {
 	// Extenders are the list of scheduler extenders, each holding the values of how to communicate
 	// with the extender. These extenders are shared by all scheduler profiles.
 	// +listType=set
-	Extenders []v1.Extender `json:"extenders,omitempty"`
+	Extenders []Extender `json:"extenders,omitempty"`
```

Comment:
Yes, `Policy` includes the _old_ type. It will be removed once `KubeSchedulerConfiguration` fully replaces `Policy`.

Reply:
Once we graduation KubeSchedulerConfiguration to GA, we can rename `v1.Extender` to `v1.DeprecatedExtender`.

Reply:
this is super subtle... can we name this more clearly to make sure we don't use the wrong struct and flip encoding of the enableHttps field?

Reply:
if we can rename the v1.Extender type later, then we can rename it now, if so, lets do it in this PR (perhaps we can using Legacy instead of Deprecated).